src/pip/_internal/commands/configuration.py

```diff
@@ -92,6 +97,7 @@ def add_options(self):
         self.parser.insert_option_group(0, self.cmd_opts)
 
     def run(self, options, args):
+        # type: (Values, List[Any]) -> int
```

Comment:
Is it not `List[str]` now?

Reply:
Fixed in https://github.com/pypa/pip/pull/8336/commits/b01eb958d08a2179a55446da7177c0c03af58fc2